src/content/docs/start-here/kafka-event.md

```diff
@@ -209,10 +209,11 @@ Example:
     "datacontenttype" : "application/json",
     "data" : {
          "metadata": {
-           "id" :  "A234-1234-1234",      
+           "id" :  "A234-1234-1234",  
+           "org_id" : "IdP's id for the organization",    
            "resource_type" : "k8s-cluster",
-           "last_reported" : "2018-04-05T17:31:00Z",
-           "workspace": "workspace name",
+           "updated_at" : "2018-04-05T17:31:00Z",
```

Comment:
Should we also include the `created_at` in here?  It is something that should be available along with the `update_at` at this point ?

The same would apply for the `deleted_at` (include the create and update)

Reply:
We could.  I am not aware of a requirement/use case that would require this in the event. Are you?  I can ask around. If not, I would suggest that we add it when needed.  As you know, adds are pretty easy to do. Modifications are much harder.  Deletes are the hardest.  Thus, if we put this in, we will be stuck with it for some time.

Reply:
Not aware of any requirement- In that case it makes sense to just keep the specific timestamp and we can evaluate later if we want to include it.